### PR TITLE
Updating Forklift to send data:write:complete to event stream after writing data

### PR DESCRIPTION
**Pull request by kennyatpillar**

### CODE REVIEW

File: apps/forklift/lib/forklift/data_writer.ex [hunk 1]
```diff
@@ -113,15 +114,16 @@ defmodule Forklift.DataWriter do
   defp do_write(data, dataset) do
     started_data = Enum.map(data, &add_start_time/1)
 
-    retry with: exponential_backoff(100) |> cap(@max_wait_time) do
+    retry with: exponential_backoff(@retry_count) |> expiry(@max_wait) do
```
Comment (bbrewer-papa-x) [inline apps/forklift/lib/forklift/data_writer.ex]: I think we misunderstood this when we edited it. We ultimately want to either set a max number of retries or a max total time so our tests can actually fail. @jdenen - the `cap` usage in this case, was to keep the exponential backoff from growing forever, right?

File: apps/andi/mix.exs [hunk 2]
```diff
@@ -54,7 +54,7 @@ defmodule Andi.MixProject do
       {:placebo, "~> 1.2", only: [:dev, :test, :integration]},
       {:plug_cowboy, "~> 2.1"},
       {:simply_validate, ">= 0.2.0"},
-      {:smart_city, "~> 3.0", override: true},
+      {:smart_city, "~> 3.11"},
```
Comment (jmorris-pillar) [inline apps/andi/mix.exs]: does this need to be specifically 3.11?  pinning to 3.0 should suffice here.

File: apps/discovery_streams/mix.exs [hunk 3]
```diff
@@ -58,7 +58,7 @@ defmodule DiscoveryStreams.Mixfile do
       {:placebo, "~> 1.2", only: [:dev, :test, :integration]},
       {:redix, "~> 0.10.2"},
       {:sweet_xml, "~> 0.6"},
-      {:smart_city, "~> 3.0"},
+      {:smart_city, "~> 3.11"},
```
Comment (jmorris-pillar) [inline apps/discovery_streams/mix.exs]: same as above

File: apps/andi/mix.exs (hunk 2 above)
Comment (kennyatpillar) [reply apps/andi/mix.exs]: We were removing the override true in places it didn't need to be and just put the current version. We'll switch this to 3.0.

File: apps/forklift/lib/forklift/data_writer.ex (hunk 1 above)
Comment (bbrewer-papa-x) [reply apps/forklift/lib/forklift/data_writer.ex]: This is fixed.

Comment (bbrewer-papa-x) [inline apps/forklift/lib/forklift/data_writer.ex]:
```suggestion
    retry with: exponential_backoff(100) |> cap(@max_wait) |> Stream.take(@retry_count) do
```

File: apps/forklift/lib/forklift/migrations.ex [hunk 4]
```diff
@@ -1,92 +1,46 @@
...
-            acc ++
-              [
-                ["RENAME", key, "old:" <> key],
-                ["EXPIRE", "old:" <> key, "600000"]
-              ]
-          end)
-
-        Redix.pipeline!(client, commands)
+  def migrate_once(completed_flag_name, event_name) do
+    Patiently.wait_for(
```
Comment (bbrewer-papa-x) [inline apps/forklift/lib/forklift/migrations.ex]: We need to put in a PR on Elsa and/or Brook to get this fixed ASAP

Comment (kennyatpillar) [reply apps/forklift/lib/forklift/migrations.ex]: Actual issue was having config: where it should have been consumer_config in the integration test. Fixed.

File: apps/forklift/lib/forklift/data_writer.ex (hunk 1 above)
Comment (kennyatpillar) [reply apps/forklift/lib/forklift/data_writer.ex]: I merged in this suggestion change. Unit tests, integration tests, and e2e all pass.

**Pull request merged**